cpp/src/neighbors/detail/vamana/vamana_build.cuh

```diff
@@ -408,9 +533,117 @@ index<T, IdxT> build(
   batched_insert_vamana<T, float, IdxT, Accessor>(
     res, params, dataset, vamana_graph.view(), &medoid_id, metric);
 
+  std::optional<raft::device_matrix<uint8_t, int64_t>> quantized_vectors;
+  if (params.codebook_prefix.size()) {
+    cuvs::neighbors::vpq_params pq_params;
```

Comment:
Question for @cjnolet - is the `vpq` API changing with 25.06 or in the future with the release of the new quantization API? If so, is CAGRA-Q having to change (which means this would have to change as well)?